lambda-notes/src/App.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom'
+import './index.css';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+import CreateNote from './components/CreateNote';
+import Notes from './components/Notes';
+import NoteCard from './components/NoteCard';
+import EditNote from './components/EditNote';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      notes: [],
+      visible: false,
+      id: null,
+    };
+  }
+  componentDidMount() {
+    axios
+      .get('https://fe-notes.herokuapp.com/note/get/all')
+      .then(response => {
+        this.setState({ notes: response.data });
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+  }
+  visible = (id) => {
```

Comment:
This is not a good name for a method. The names you choose should be helpful for future viewers of your code.